src/GlobalStyles.js

```diff
@@ -29,7 +29,7 @@ button,
 textarea, 
 p {
   font-size: inherit;
-  font-family: Helvetica Neue;
+  font-family: 'Helvetica Neue';
```

Comment:
You actually only need to add the `font-family` once in the `body` and it should then be inherited by the children.